buffer/src/main/java/io/netty/buffer/DuplicatedAbstractByteBuf.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 The Netty Project
+ * Copyright 2016 The Netty Project
```

Comment:
File has been updated, so the year should be updated. We could say 'copyright 2015, 2016 ...' but I think the last modified year is enough.